app/validators/routing/types.py

```diff
@@ -60,6 +60,11 @@ def resolve_value_source_json_type(value_source, schema):
         answer_type = schema.answers_with_context[answer_id]["answer"]["type"]
         return ANSWER_TYPE_TO_JSON_TYPE[answer_type]
 
+    if source == "grand_calculated_summary":
```

Comment:
A unit test needs adding to `test_types.py`.

Reply:
done

Reply:
Resolving on behalf of @MebinAbraham 👍